extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-sqlserver/src/main/java/org/apache/guacamole/auth/sqlserver/conf/SQLServerEnvironment.java

```diff
@@ -295,4 +295,36 @@ public boolean enforceAccessWindowsForActiveSessions() throws GuacamoleException
                 true);
     }
 
+    @Override
+    public boolean shouldUseBatchExecutor() {
```

Comment:
This change isn't required to fix the problem with the batch executor, but it is awfully handy for testing using self-hosted SQL Server instances with self-signed certificates.